docs/2_getting-started.md

```diff
@@ -0,0 +1,198 @@
+2020-06-05 12:42:03 INFO __main__ verbosity: None; log level: INFO; handler level: INFO
+2020-06-05 12:42:03 INFO __main__ Creating kubernetes client with config /Users/chaos/.kube/config (path found for backwards compatibility)
+2020-06-05 12:42:03 INFO k8s_client Initializing with config: /Users/chaos/.kube/config
+2020-06-05 12:42:03 INFO __main__ No cloud driver - some functionality disabled
+2020-06-05 12:42:04 INFO __main__ Using stdout metrics collector
+2020-06-05 12:42:04 INFO __main__ Starting the UI server (0.0.0.0:8000)
+2020-06-05 12:42:04 INFO __main__ STARTING AUTONOMOUS MODE
+2020-06-05 12:42:04 INFO scenario.Hello chaos! Starting scenario 'Hello chaos!' (3 steps)
+2020-06-05 12:42:04 INFO action_nodes_pods.Hello chaos! Matching 'namespace' {'namespace': 'kube-system'}
+2020-06-05 12:42:04 INFO action_nodes_pods.Hello chaos! Matched 37 pods in namespace kube-system
+2020-06-05 12:42:04 INFO action_nodes_pods.Hello chaos! Filtered set length: 1
+2020-06-05 12:42:04 INFO action_nodes_pods.Hello chaos! Pod killed: [pod #32 name=kube-state-metrics-7b4944dfbb-zrlrz namespace=kube-system containers=1 ...]
+2020-06-05 12:42:04 INFO scenario.Hello chaos! Sleeping for 30 seconds
+2020-06-05 12:42:34 INFO action_nodes_pods.Hello chaos! Matching 'namespace' {'namespace': 'kube-system'}
+2020-06-05 12:42:35 INFO action_nodes_pods.Hello chaos! Matched 37 pods in namespace kube-system
+2020-06-05 12:42:35 INFO action_nodes_pods.Hello chaos! Filtered set length: 37
+2020-06-05 12:42:35 INFO scenario.Hello chaos! Scenario finished
+```
+
+We just verified experimentally, that the `Kubernetes` cluster was able to successfully restart a pod within 30 seconds. Now, what would happen if you changed the wait to 1 second?
+
+```yaml
+...
+  - wait:
+      seconds: 1
+```
+
+Most likely this:
+
+```
+...
+2020-06-05 12:53:26 INFO action_nodes_pods.Hello chaos! Pod killed: [pod #14 name=kube-apiserver-metrics-pkh6n namespace=kube-system containers=1 ...]
+2020-06-05 12:53:26 INFO scenario.Hello chaos! Sleeping for 1 seconds
+2020-06-05 12:53:27 INFO action_nodes_pods.Hello chaos! Matching 'namespace' {'namespace': 'kube-system'}
+2020-06-05 12:53:27 INFO action_nodes_pods.Hello chaos! Matched 37 pods in namespace kube-system
+2020-06-05 12:53:27 INFO action_nodes_pods.Hello chaos! Initial set length: 37
+2020-06-05 12:53:27 INFO action_nodes_pods.Hello chaos! Filtered set length: 37
+2020-06-05 12:53:27 ERROR action_nodes_pods.Hello chaos! Expected pod in state 'Running', got 'ContainerCreating' ([pod #14 name=kube-apiserver-metrics-f98lw namespace=kube-system containers=1 ...])
+2020-06-05 12:53:27 WARNING scenario.Hello chaos! Step returned failure {'podAction': {'matches': [{'namespace': 'kube-system'}], 'actions': [{'checkPodState': {'state': 'Running'}}]}}. Finishing scenario early
+2020-06-05 12:53:27 ERROR policy_runner Exiting early
+2020-06-05 12:53:27 ERROR __main__ Policy runner finishes with an error
+```
+
```

Comment:
I would add here, that autonomous mode is the most used/more configurable mode... Or something like that.

Reply:
Done